include/vcpkg/base/chrono.h

```diff
@@ -48,10 +48,9 @@ namespace vcpkg
 
         ElapsedTimer() noexcept;
 
-        ElapsedTime elapsed() const
-        {
-            return ElapsedTime(clock::now() - time_point(duration(this->m_start_tick.load())));
-        }
+        time_point start() const { return time_point(duration(this->m_start_tick.load())); }
```

Comment:
Note that this returns `high_resolution_clock::time_point` but you are trying to pass it to things that say `system_clock::time_point`.